src/main/resources/assets/splatcraft/lang/it_it.json

```diff
@@ -409,8 +418,27 @@
   "commands.colorscores.list.count": "Mostrando il criterio del colore %s:",
   "commands.colorscores.list.entry": "%s: visibile come '%s'",
 
+  "commands.stage.add.success": "Aggiunto lo scenario [%s]",
+  "commands.stage.remove.success": "Rimosso lo scenario [%s]",
+  "commands.stage.setting.success.default": "Resettato %s dello scenario [%s]",
+  "commands.stage.setting.success": "L'impostazione %s dello scenario [%s] è ora: %s",
+  "commands.stage.setting.get.default": "L'impostazione %s dello scenario [%s] è attualmente impostata a: default",
+  "commands.stage.setting.get": "L'impostazione %s dello scenario [%s] è attualmente impostata a: %s",
+  "commands.stage.setting.area.success": "Impostato il vertice %s dello scenario [%s] a: (%s, %s, %s)",
+  "commands.stage.setting.area.get": "Il vertice %s dello scenario [%s] è attualmente impostato a: (%s, %s, %s)",
+  "commands.stage.teams.set.success": "Taggati %s blocchi nello scenario %s come appartenenti alla squadra [%s]",
+  "commands.stage.teams.remove.success": "Rimossa la squadra [%s] dallo scenario %s e l'appartenenza a %s blocchi",
+  "commands.stage.teams.get.success": "La squadra [%s] dello scenario %s è attualmente impostata a: %s",
+  "commands.stage.warp.success": "Teletrasportati %s giocatore/i allo scenario [%s]",
+
   "arg.colorCriterion.notFound": "Nessun criterio geristrato per %s",
   "arg.inkColor.notFound": "Colore sconosciuto: %s",
+  "arg.stageWarp.noSpawnPads": "No available Spawn Pads were found on stage [%s]", # TRANSLATE ME
+  "arg.stageWarp.noPlayers": "No players were found with a Spawn Pad matching their ink color on stage [%s]", # TRANSLATE ME
+  "arg.stage.notFound": "Stage [%s] does not exist", # TRANSLATE ME
+  "arg.stage.alreadyExists": "Stage [%s] already exists", # TRANSLATE ME
+  "arg.stageTeam.notFound": "Unknown team %s for stage [%s]", # TRANSLATE ME
+  "arg.stageSetting.notFound": "Setting %s does not exist", # TRANSLATE ME
```

Comment:
don't forget to remove the `# TRANSLATE ME`s

Reply:
they're now gone.